src/awake.hpp

```diff
@@ -439,7 +439,8 @@ enum {
 #define PRF_NO_WEATHER                   64
 #define PRF_DISABLE_XTERM                65
 #define PRF_COERCE_ANSI                  66
-#define PRF_MAX                          67
+#define PRF_VISIBLE                      67
```

Comment:
**Please test your code before you submit a pull request.** Whenever you add a preference flag, you need to add it to the constants block for preference flags, and I've configured the game to not even boot up before this step is complete. You'll get the following error message if you launch the game in its submitted state:
```
Aug  1 07:03:14 :: Error: You added a prf flag but didn't add it to preference_bits in constants.cpp (or forgot a comma)!
Process 27388 exited with status = 54 (0x00000036)
```

Also, PRF_VISIBLE isn't easy to decipher what it is when reading the code-- it sounds like it's related to staff visibility. Let's rename the definition to PRF_HIDDEN_FROM_WHERE or something to make it self-documenting.

Reply:
Good to know. At the moment, I wanted to submit a draft, but didn't know if the code was even going to get merged. Just wanted it out of my head.

I'll make those changes anyway, good practice.